Client/core/CMainMenu.cpp

```diff
@@ -221,6 +233,44 @@ CMainMenu::CMainMenu(CGUI* pManager)
     m_pLatestNews->SetProperty("InheritsAlpha", "False");
     m_pLatestNews->SetVisible(false);
 
+    float offsetX = 0.01f;
+    size_t i = 0;
```

Comment:
Is this line even used?

Reply:
`offsetX` is used on lines `249` and `271` for position calculation.
`socialsIndex` is used on line `244`